stackslib/src/net/stackerdb/sync.rs

```diff
@@ -246,6 +249,8 @@ impl<NC: NeighborComms> StackerDBSync<NC> {
             .collect();
 
         schedule.sort_by(|item_1, item_2| item_1.1.len().cmp(&item_2.1.len()));
+        schedule.reverse();
+
```

Comment:
I think this instance was in the correct order before: when making requests _for_ data, you want to request in rarest-first order, which means ordering by which chunks have the least number of servers (`schedule` is a list of who _has_ the data). In `make_chunk_push_schedule`, `schedule` is a list of who _needs_ the data, so rarest-first order means ordering by the longest lists (where a `reverse()` call makes the order correct)